google/calendar/tools/event.py

```diff
@@ -417,6 +413,47 @@ def quick_add_event(service):
         raise Exception(f"Exception quick adding event to calendar {calendar_id}: {e}")
 
 
+def _get_recurrence_list(recurrence: str) -> list:
+    """
+    Parse a string into a list of recurrence rules.
+
+    Args:
+        recurrence (str): The recurrence string to parse
+
+    Raises:
+        ValueError: If the recurrence string is not a valid JSON array of strings, where each string is an RRULE, EXRULE, RDATE, or EXDATE line as defined by the RFC5545.
+        ValueError: If the recurrence string is not a valid recurrence rule syntax.
+
+    Returns:
+        list: A list of recurrence rules
+    """
+    try:
+        recurrence_list = json.loads(recurrence)
+    except json.JSONDecodeError:
+        try:
+            fixed_recurrence = recurrence.encode().decode(
```

Comment:
try to fix the input by treating all the \ escape sequences like they were actual escaped characters again.